bundle/cert-manager-operator/0.15.0-alpha.0/cert-manager-operator.v0.15.0-alpha.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,90 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - serviceAccountName: cert-manager-operator
+        rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
```

Comment:
We really should look to reduce this at some point.. for now though LGTM :)

Reply:
Agreed https://github.com/jetstack/cert-manager-olm/issues/3